.classpath

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="lib"/>
+	<classpathentry kind="src" path="test"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/5"/>
+	<classpathentry kind="lib" path="C:/Users/salem/eclipse-workspace/NovaTechLMSSQL/lib/mysql-connector-java-5.1.47.jar"/>
```

Comment:
This definitely needs to be changed from an absolute path to one relative to the project root.

Reply:
So, you mean something like lib/mysql-connector-java-5.1.47.jar?

Reply:
Right. You used relative paths for the other libraries you added to `.classpath`, but this one won't work on anyone else's computer as it stands.

Reply:
You are right. I changed it and pushed it.